packages/analytics/analytics-metric-provider/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Change Log
```

Comment:
nit: I'd recommend deleting this file completely and it will be auto-generated

Reply:
Ah, I wasn't sure -- done.